plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/list_investigations/schema.py

```diff
@@ -247,37 +228,36 @@ class ListInvestigationsOutput(insightconnect_plugin_runtime.Output):
           "title": "Title",
           "description": "Investigation title",
           "order": 12
-        }
-      },
-      "required": [
-        "created_time",
-        "disposition",
-        "last_accessed",
-        "organization_id",
-        "priority",
-        "rrn",
-        "source",
-        "status",
-        "title"
-      ],
-      "definitions": {
-        "assignee": {
-          "type": "object",
-          "title": "assignee",
-          "properties": {
-            "email": {
-              "type": "string",
-              "title": "Email",
-              "description": "The email of the assigned user",
-              "order": 1
-            },
-            "name": {
-              "type": "string",
-              "title": "Name",
-              "description": "The name of the assigned user",
-              "order": 2
-            }
-          }
+        },
+        "required": [
+          "created_time",
+          "disposition",
+          "last_accessed",
+          "organization_id",
+          "priority",
+          "rrn",
+          "source",
+          "status",
+          "title",
+          "investigations"
```

Comment:
as investigations is not a property in this object, it will cause an error as it will not be present